Extensions/pokes.css

```diff
@@ -1,73 +1,215 @@
 }
 
-#xkit-loading_pokemon {
+/*
+ * Pokémon list style 
+*/
+.xkit-pokes-lightbox {
+	position: fixed;
+	top: 0px;
+	bottom: 0px;
+	right: 0px;
+	left: 0px; 
+	z-index: 15000;
+	overflow: hidden;
+	background-color: rgba(25,25,25,.92);
+}
+.gender {
+	display: none;
+	float: right;
+	font-family: "segoe ui symbol";
+	font-size: 20px;
+}
+.nickname {
+	display: none;
+	font-weight: 700;
+	font-size: 25px;
+}
+.nickname::before {
+	content: "";
+	font-family: "tumblr-icons";
+	float: left;
+	display: none;
+}
+.species {
+	display: none;
+	font-size: 16px;
+	margin-top: 3px;
+}
+#xkit-pokes-nickinput {
+	font-size: 20px;
+}
+.xkit-pokes-pc {
+	max-width: 800px;
+	border-radius: 40px;
+	height: 600px;
+	background: #EFEFE7;
+	position: relative;
+	margin: 50px auto;
+}
+.xkit-pokes-pc-pokemon {
+	height: 350px;
+	width:85%;
+	padding: 10px;
+	margin: 50px;
+	position: absolute;
+	border: 4px solid #6B94B5;
+	border-radius: 20px;
+	background: #B6E4EF;
+}
+.xkit-pokes-pc-info {
+	position: absolute;
+	background: #fff;		
+	height: 90px;
+	width: 85%;
+	padding: 10px;
+	margin: 440px 50px 50px 50px;
+	border: 4px solid #6B94B5;
+	border-radius: 20px;
+	-webkit-filter: grayscale(30%);
+	filter: grayscale(0%);
+}
+.xkit-pokes-pc-info .gender[data-pokegender="male"]::after {
+	content: " \2642";
+	color: #6CA0DC;
+	font-family: "segoe ui symbol";
+}
+.xkit-pokes-pc-info .gender[data-pokegender="female"]::after {
+	content: " \2640";
+	font-family: "segoe ui symbol";
+	color: #FF69B4;
+}
+.xkit-pokes-pc-info .gender[data-pokegender="genderless"]::after {
+	content: " \26AA";
+	color: #9370DB;
+}
+.xkit-pokes-pc-info .capture_rate {
```

Comment:
That is, unless you mean which Pokémon are already caught, in which case the naming is very unclear (as it implies rarity of a singular Pokémon).